project_national/fact_sheets_category_4.yml

```diff
@@ -0,0 +1,276 @@
+            - HVAC Cooling Efficiency|Room AC, EER 10.7
+
+  - upgrade_name: ENERGYSTAR Central AC
+    options:
+      - &energystar_central_ac
+        option: HVAC Cooling Efficiency |AC, SEER 15.2
+        apply_logic:
+          - or:
+            - HVAC Cooling Efficiency|AC, SEER 8
+            - HVAC Cooling Efficiency|AC, SEER 10
+            - HVAC Cooling Efficiency|AC, SEER 13
+            - HVAC Cooling Efficiency|AC, SEER 14
+            - HVAC Cooling Efficiency|AC, SEER 15
+
+  - upgrade_name: HPWH
+    options:
+      - &water_heater_fuel_type
+        option: Water Heater Fuel|Electricity
+      - &water_heater_efficiency_electric_heat_pump_50_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 50 gal, 3.45 UEF
+        apply_logic:
+          - or:
+            - Occupants|0
+            - Occupants|1
+            - Occupants|2
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+      - &water_heater_efficiency_electric_heat_pump_66_gal_3_pt_35_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 66 gal, 3.35 UEF
+        apply_logic:
+          - or:
+            - Occupants|3
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+      - &water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 80 gal, 3.45 UEF
+        apply_logic:
+          - or:
+            - Occupants|4
+            - Occupants|5
+            - Occupants|6
+            - Occupants|7
+            - Occupants|8
+            - Occupants|9
+            - Occupants|10+
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+
+  - upgrade_name: 96% AFUE Natural Gas Furnace
+    options:
+      - &Furnace_fuel_type
+        option: HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+        apply_logic:
+          - and: 
+            - or:
+              - Heating Fuel|Natural Gas
+              - Water Heater Fuel|Natural Gas # Do we need more to represent any natural gas usage?
+            - or:
+              - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace
+              - HVAC Heating Type And Fuel|Propane Fuel Furnace
+              - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace
+      - &96%AFUE_furnace
+        option: HVAC Heating Efficiency|Fuel Furnace, 96% AFUE
+        apply_logic:
+          - and: 
+            - or:
+              - Heating Fuel|Natural Gas
+              - Water Heater Fuel|Natural Gas
+            - or:
+              - HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+              - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace
+              - HVAC Heating Type And Fuel|Propane Fuel Furnace
+              - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace
+            - or:
```

Comment:
I think this is where you could use `apply_package_logic` because you want to change `HVAC Heating Type and Fuel` and `HVAC Heating Efficiency` for the same downselection.

Reply:
I did not use apply_package_logic before. Could you let me know the reference or any yml file that use apply_package_logic?

Reply:
Example: https://github.com/NREL/resstock/blob/e6780b5e9f73d8bcf298e46fb3b36300c54a0ce0/project_chicago/chicago_packages_utility.yml#L394-L398

Reply:
Done